AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift

```diff
@@ -14,15 +14,53 @@
 //  Copyright © 2018 Amahi. All rights reserved.
 //
 
+//import Foundation
+//
+//struct ApiConfig {
+//
+//    static let BASE_URL =                       "https://example.com"
+//    static let PROXY_URL =                       "https://example.com"
+//
+//    private static let CLIENT_ID =              "C-I-D"
+//    private static let CLIENT_SECRET =          "C-S-T"
+//
+//    static func oauthCredentials(username: String, password: String) -> [String : String] {
+//
+//        let parameters =                          ["client_id": CLIENT_ID,
+//                                                   "client_secret": CLIENT_SECRET,
+//                                                   "username" : username,
+//                                                   "password" : password ]
+//
+//        fatalError("You may need to get dev credentials to be able to properly login into amahi.org." +
+//            "Get them from support at amahi dot org. Then remove this line")
+//
+//        return parameters
+//    }
+//}
+//
+//  ********************************************************
+//  *************   DO NOT COMMMIT THIS FILE   *************
+//  *************         DO NOT SHARE IT      *************
+//  ************* It Has Sensitive Credentials *************
+//  ********************************************************
+//
+//  AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
+//
+//  AmahiAnywhere
+//
+//  Copyright © 2018 Amahi. All rights reserved.
+//
+
 import Foundation
 
 struct ApiConfig {
     
-    static let BASE_URL =                       "https://example.com"
-    static let PROXY_URL =                       "https://example.com"
+    static let BASE_URL =                       "https://api.amahi.org"
+    static let PROXY_URL =                      "https://pfe.amahi.org"
+    
+    private static let CLIENT_ID =              "0714493ef3f647209dfe0574fbc3b426220714aaedfd6c007cd54f33e4a0d4bb"
```

Comment:
You should not have pushed this file to your PR. Now out secret Api Configurations are out. 
Please delete this PR so it is not accessible to the public.

Reply:
sure sorry for this

Reply:
how to delete this pr?

Reply:
git checkout devansh_gsoc
git reset --soft HEAD~1
git checkout AmahiAnywhere/AmahiAnywhere/Data/Remote/ApiConfig.swift
git add .
git commit -m "commit name"
git push -f origin devansh_gsoc

Reply:
ok let me try this, if it work I will update you, how do you have so much knowledge its cool tbh

Reply:
This is not working :(

Reply:
make sure you are on same branch.